blog/_posts/2021-10-25-tilt-news-october-2021.md

```diff
@@ -0,0 +1,97 @@
+*The Tilt team hanging out in our KubeCon booth. From left to right: Nick, Milas, Lizz, Dan, and Surbhi*
+
+## Remocal Development
+
+We heard a new phrase at KubeCon from InfluxData's Developer experience team: “remocal” dev.
+
+Here's the idea: if you think about dev as local or remote, you misunderstand the world we live in. Modern dev environments are “remocal”—they figure out how your local machine fits into a dev environment with remote components.
+
+We've been thinking more and more about this problem. And experimenting! And writing posts about what we learned, including:
+
+- [Using kubefwd to connect Kubernetes to your local machine](https://blog.tilt.dev/2021/09/09/kubefwd-operator.html)
+- [Using ngrok to connect your local machine to a public URL](https://blog.tilt.dev/2021/09/21/ngrok-operator.html)
+- [Using Tailscale to connect your laptop to a dev docker](https://blog.tilt.dev/2021/10/11/old-school-remote-dev-clusters.html)
+
+![Tilt UI resourve view with custom button that reads 'start ngrok'](/assets/images/tilt-news-october-2021/ngrok1.jpg)
+
+## The Future of Docker
+
+The Docker licensing changes have kicked off a larger conversation about the place of Docker in local dev environments. Is this the right tool? How can we use it better? Should we be considering alternatives?
+
+We've written some posts to help people think through this:
+
+- [Improving Docker performance](https://blog.tilt.dev/2021/09/13/docker-does-not-mean-slow.html)
+- [Swapping Rancher Desktop for Docker Desktop](https://blog.tilt.dev/2021/09/07/rancher-desktop.html)
+
+## Kubernetes Cluster API
+
+In other Kubernetes news, Cluster API is now officially production-ready!
+
+Cluster API is a Kubernetes project that enables declarative management for Kubernetes, using APIs to easily create, configure, and update clusters. It is an end-to-end approach that can simplify the repetitive tasks of the Kubernetes lifecycle, while maintaining consistency and repeatability across a unified infrastructure.
+
+They are also the biggest open-source project that uses Tilt. Check out [their developer guide](https://cluster-api.sigs.k8s.io/developer/tilt.html) to see how they are leveraging Tilt for rapid iterative development.
+
+## Tilt News
+
+We were thrilled to see Tilt mentioned in three blogposts:
+
+Kevin Lindsay, Full Stack Web Developer at Surge and OpenFaas community member, writes about using Tilt to bring smart rebuilds and live updates to [OpenFaaS functions](https://www.openfaas.com/blog/tilt/).
+
+Archers.ai’s mission is to help its users track cloud resources for both cost-optimization and tax purposes. They care a lot about the ability to track the cost of dev environments, and they see tools like Tilt as essential for helping them do that. In this blog post, Jason Burt, Product at Archera.ai explains how they are using Tilt to [accelerate their product feedback loop](https://archera.ai/blog/how-we-used-tilt/).
```

Comment:
I'm not sure this is a good explanation of what Archera does. Maybe grab some language from their site? (or just ask them TBH)